NEWS

```diff
@@ -25,20 +25,39 @@ issues]].
   - Add various Chinese phrases involving exceptions of braille
     patterns
   - Add a math symbol ◊ (⠫⠓)
+- Improvements to the Russian literary (uncontracted and contracted)
+  braille thanks to Andrey Yakuboy.
+  - If the dot sign (.) is after digits at the beginning of a string,
+    the braille dot 6 isn't inserted between the last digit and the
+    dot. This is necessary for items' numbers of numbered lists.
+  - Back translation for French letters has been disabled because of
+    conflicts with punctuations.
+  - Back translation for Latin (mostly English and German) letters has
+    been improved.
+  - The definition of the 7/8 fraction (U+215E) has been fixed.
+  - Placement of spaces around math symbols has been improved
+    according to the Russian braille rules.
+  - New display table (~ru-brf.dis~) has been added. It adds support
+    of Braille ASCII for Russian and other Cyrillic tables.
```

Comment:
I'm not sure about the new display table. For one, it would've been better if this was done in a separate commit. Second, I would expect that ru-brf.dis would either include en-us-brf.dis, or that en-us-brf.dis would have been updated. A more formal explanation of what is meant by "BrailleAscii", including a reference, as is done in en-us-brf.dis, would also have been nice I think.

Finally, I think dropping virtual dots is something that could be handled on another level than the display table. In fact with the Java bindings you already have the possibility to do this (see the [`DisplayTableFromTable`](https://github.com/liblouis/liblouis-java/blob/28c6d8ce51f1158b196e0eece62ca02653ff1373/src/main/java/org/liblouis/DisplayTable.java#L336)). A similar option could be added to Liblouis itself.